2996_reflection/reflection.md

```diff
@@ -5386,7 +5391,7 @@ consteval bool is_data_member_spec(info r);
 
 [#]{.pnum} *Constant When*: Letting `@$r$~$K$~@` be the `$K$`^th^ reflection value in `mdescrs`,
 
-- `class_type` represents an incomplete class type,
+- `class_type` represents an incomplete class type `$C$`. One of the points in the evaluation context ([expr.const]) belongs to the same module to which `$C$` is attached,
```

Comment:
Similar "one of the points" comment.

Reply:
Made the same adjustment here; done. The adjustment is minor, but let me know if it reads any better.